### PR TITLE
Add CDI manifests to metalkube dev-scripts

### PR DESCRIPTION
**Pull request by rthallisey**

### CODE REVIEW

**Comment by derekhiggins** [comment]
Build SUCCESS, see build http://10.8.144.11:8080/job/dev-tools/283/

File: manifests/README_cnv.md [hunk 1]
```diff
@@ -37,7 +37,26 @@ Apply this config https://github.com/kubevirt/demo/blob/master/manifests/vm.yaml
...
 
-TBD
+CDI provides the DataVolume CRD to use with VMs.  Enable 'DataVolumes' feature
+gate in the kubevirt configmap to use them.
+
+```bash
+VERSION=v1.6.0
+curl -Lo 8_cnv_cdi_operator.yaml https://github.com/kubevirt/containerized-data-importer/releases/download/$VERSION/cdi-operator.yaml
+curl -Lo 9_cnv_cdi_cr.yaml https://github.com/kubevirt/containerized-data-importer/releases/download/$VERSION/cdi-operator-cr.yaml
+cat > 12_cnv_kubevirt_config.yaml <<EOY
```
Comment (booxter) [inline manifests/README_cnv.md]: should we make this configmap a manifest automatically applied?

Comment (rthallisey) [reply manifests/README_cnv.md]: yes.  Adding it.

**Comment by aglitke** [comment]
@annastopel fyi

**Comment by hardys** [comment]
Ok tested this and I see the CDI pods running, so merging and we can iterate on any further changes as follow-up PRs, thanks!

**Pull request merged**